app/Http/Controllers/AuthorController.php

```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
 
         return response()->json(['id' => $author->id]);
     }
+    public function destroy($id)
+    {
+
+        Author::destroy($id);
+        /*
+        !!No se encuentran los metodos!!
+        $author= Author::find($id);
+        $author->delete();
+        */
+
+
+        return "ok";
+    }
+
+    public function index()
+    {
+        $author=Author::all();
+
+
+        return $author;
```

Comment:
We should return something like `response()->json($authors);`.

And the name should be $authors.